controller/workitem_blackbox_test.go

```diff
@@ -1806,6 +1806,11 @@ func assertSingleWorkItem(t *testing.T, createdWI app.WorkItemSingle, fetchedWI
 	assert.NotNil(t, fetchedWI.Data.Links.Self)
 	assert.NotNil(t, fetchedWI.Data.Relationships.Creator.Data.ID)
 	assert.NotNil(t, fetchedWI.Data.Relationships.BaseType.Data.ID)
+	relatedLink := fmt.Sprintf("/%s/labels", fetchedWI.Data.ID)
+	assert.NotNil(t, fetchedWI.Data.Relationships.Labels)
```

Comment:
`require.NotNil()` here, too

Reply:
done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/d669b65ccc5f6d78c3bc11494e6065ee6341559c